UnityClient/Assets/Scripts/Components/UI/Background.cs

```diff
@@ -0,0 +1,24 @@
+using ROIO;
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Background : MonoBehaviour {
+
+    private const string LOGIN_BACKGROUND_IMAGE = "bgi_temp.bmp";
+    private const int LOADING_SCREENS_LENGTH = 10;
+
+    private static RawImage RawImage;
```

Comment:
Is this needed? you never assign a value to it

Reply:
I have removed this file since we are using the RawImage extensions only